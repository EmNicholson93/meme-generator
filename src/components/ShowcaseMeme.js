import React from 'react';
import PropTypes from 'prop-types';

function ShowcaseMeme({ head, imageSrc, foot }) {
  return (
    <>
      <p>{head}</p>
      <img src={imageSrc} />
      <p>{foot}</p>
    </>
  );
}

ShowcaseMeme.propTypes = {
  head: PropTypes.string.isRequired,
  imageSrc: PropTypes.string.isRequired,
  foot: PropTypes.string.isRequired
};

export default ShowcaseMeme;

