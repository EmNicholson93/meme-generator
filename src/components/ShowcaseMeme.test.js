import React from 'react';
import { shallow } from 'enzyme';
import ShowcaseMeme from './ShowcaseMeme';

describe('ShowcaseMeme component', () => {
  it('renders ShowcaseMeme', () => {
    const wrapper = shallow(<ShowcaseMeme head="hello" imgSrc="yolo" foot="feet" />);
    expect(wrapper).toMatchSnapshot();
  });
});