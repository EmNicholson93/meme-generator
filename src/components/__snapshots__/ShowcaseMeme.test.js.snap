// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowcaseMeme component renders ShowcaseMeme 1`] = `
<Fragment>
  <p>
    hello
  </p>
  <img />
  <p>
    feet
  </p>
</Fragment>
`;
